### PR TITLE
try to fix body being weird (width)

### PR DESCRIPTION
**Pull request by NovaSagittarii**

### CODE REVIEW

**Comment by NovaSagittarii** [comment]
https://stackoverflow.com/a/14270832

i love google

**Comment by NovaSagittarii** [comment]
plz merge, ibeg of u . it'll fix EVWERYTHING LITERALLY plzzzz :pray: :moyai: 🙃

**Comment by SafeDuck** [comment]
Sorry my bad...

**Review by SafeDuck — APPROVED** [review]
good

File: style.css [hunk 1]
```diff
@@ -2,7 +2,12 @@
     color: white;
     }
 
+html, body {
+  overflow-x:hidden 
+}
```
Comment (NovaSagittarii) [inline style.css:7]: https://stackoverflow.com/a/14270832

**Pull request merged**